wayshot/src/wayshot.rs

```diff
@@ -66,15 +66,11 @@ fn main() -> Result<()> {
         return Ok(());
     }
 
-    let image_buffer = if let Some(slurp_region) = cli.slurp {
-        let slurp_region = slurp_region.clone();
+    let image_buffer = if cli.slurp {
         wayshot_conn.screenshot_freeze(
             Box::new(move || {
                 || -> Result<LogicalRegion> {
-                    let slurp_output = Command::new("slurp")
-                        .args(slurp_region.split(' '))
-                        .output()?
-                        .stdout;
```

Comment:
Why change this behaviour? This is removing the ability to pass arguments to slurp.

Reply:
slurp only accept '-d' to select screen, but -d cannot be passed into, and it is useless

Reply:
![image](https://github.com/waycrate/wayshot/assets/60290287/f6029484-f0f1-4e83-b7f5-83c1ea0aaf7e)
all the argument is like `-d` `-w`, how can you pass the argument into it? I have try send it , but it can never be received

Reply:
`cargo run -- --slurp '\-d'`
this command will work

but '-d' will never work